src/v/kafka/server/connection_context.cc

```diff
@@ -340,7 +340,6 @@ ss::future<> connection_context::process_next_response() {
         }
 
         auto msg = response_as_scattered(std::move(r));
-        _rs.probe().add_bytes_sent(msg.size());
```

Comment:
What is this change? I think this was removed recently. Perhaps it snuck into your tree and you need a rebase.

Reply:
That's what I'm thinking about tbh! Because I also didn't touch autotune.md, except the "We recommend having at least 1GB of memory." text.